packages/circuit-ui/styles/style-mixins.spec.tsx

```diff
@@ -36,24 +36,24 @@ import {
 } from './style-mixins';
 
 describe('Style helpers', () => {
-  const red = (theme: Theme) => css`
-    color: ${theme.colors.r500};
+  const byte = (theme: Theme) => css`
+    margin: ${theme.spacings.byte};
   `;
-  const green = (theme: Theme) => css`
-    color: ${theme.colors.g500};
+  const kilo = (theme: Theme) => css`
+    color: ${theme.spacings.kilo};
   `;
   const purple = css`
     color: rebeccapurple;
   `;
 
   describe('cx', () => {
     it('should call each style function with the theme', () => {
-      const actual = create(<div css={cx(red, green)} />);
+      const actual = create(<div css={cx(byte, kilo)} />);
 
       expect(actual).toMatchInlineSnapshot(`
         .circuit-0 {
-          color: #D23F47;
-          color: #8CC13F;
+          margin: 8px;
+          color: 12px;
```

Comment:
...and in case you want to apply the above from the GH UI:

```suggestion
          margin: 12px;
```